internal/config/config.go

```diff
@@ -33,7 +34,9 @@ type MQTTConfig struct {
 }
 
 type StandbyConfig struct {
-	BackupFile string `yaml:"backup_file" default:"plan.json"`
+	BackupFile      string        `yaml:"backup_file" default:"plan.json"`
+	CheckInterval   time.Duration `yaml:"check_interval" default:"60s"`
+	OutageThreshold time.Duration `yaml:"outage_interval" default:"180s"`
```

Comment:
Please note: yaml tag is still saying `interval` instead of `threshold`.

Reply:
🤦‍♀️  thanks